warp10/src/main/sh/package.sh

```diff
@@ -28,60 +30,66 @@ WARP10_HOME=warp10-${VERSION}
 ARCHIVE=${WARP_ROOT_PATH}/archive
```

Comment:
check that WARP_ROOT_PATH does exist since a dangerous rm -rf is performed afterwards.

Reply:
done